aws/resource_aws_launch_template.go

```diff
@@ -57,8 +57,16 @@ func resourceAwsLaunchTemplate() *schema.Resource {
 			},
 
 			"default_version": {
```

Comment:
We should consider validating configuration values, especially to make the logic easier later on 👍 

```go
ValidateFunc: validation.IntAtLeast(1),
```